Importer/Interpreter/Field/BooleanFieldInterpreter.php

```diff
@@ -32,6 +47,20 @@ public function interpret(FieldConfigurationInterface $configuration, $value)
             return null;
         }
 
-        return !in_array($value, $this->falseRepresentations, true);
+        if (is_bool($value)) {
+            return $value;
+        }
+
+        $value = strtolower($value);
```

Comment:
What if it's an array? Or object?
And floats will be converted to strings. Why?